### PR TITLE
add ChaosDAO, GMorDIE, Stellaswap

### PR DESCRIPTION
**Pull request by spazcoin**
* add ChaosDAO repos
* add GMorDIE parachain
* add Stellaswap to Moonbeam
* add Tide Labs. (Not on DOT/KSM relay, but substrate/rust based)

### CODE REVIEW

**Comment by spazcoin** [comment]
FYI @x676f64 @arrudagates @tomen

**Comment by helloitsbirdo** [comment]
A+

File: data/ecosystems/p/polkadot.toml [hunk 1]
```diff
@@ -149,6 +152,7 @@ sub_ecosystems = [
   "Spheroid Universe",
   "Stafi",
   "Stafi Protocol",
+  "Stellaswap",
```
Comment (jubos) [inline data/ecosystems/p/polkadot.toml:155]: Moonbeam is a subecosystem of Polkadot already so don't need this here.

File: data/ecosystems/p/polkadot.toml [hunk 2]
```diff
@@ -161,6 +165,7 @@ sub_ecosystems = [
   "Tea Project",
   "Ternoa",
   "Tidal Finance",
+  "Tide Labs",
```
Comment (jubos) [inline data/ecosystems/p/polkadot.toml:168]: Tide Labs shouldn't be part of Polkadot if they are just using substrate.  It is similar to chains using Tendermint.

**Review by jubos — CHANGES_REQUESTED** [review]

**Comment by spazcoin** [comment]
ok @jubos make your suggested corrected changes

**Comment by jubos** [comment]
Merged this, thank you!

**Pull request closed without merging**